### PR TITLE
Added recipe for geonum

### PR DESCRIPTION
**Pull request by jgliss**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jgliss** [comment]
@conda-forge/staged-recipes Will this be merged soon?

**Comment by ocefpaf** [comment]
> @conda-forge/staged-recipes Will this be merged soon?

Instead of asking if it will be merged soon please point to the team that the CIs are passing and request a review. See https://github.com/conda-forge/staged-recipes/#12-my-pull-request-passes-all-checks-but-hasnt-received-any-attention-how-do-i-call-attention-to-my-pr--what-is-the-customary-amount-of-time-to-wait

File: recipes/geonum/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - srtm.py
+
+test:
+  imports:
+    - geonum
+    - geonum.test
+
+about:
+  home: https://github.com/jgliss/geonum
+  license: GNU General Public v3 or later (GPLv3+)
```
Comment (ocefpaf) [inline recipes/geonum/meta.yaml]: Please use the short SPDX string: `GPL-3.0`.

File: recipes/geonum/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f393389a0c22664613075cd22e784aee264e7ea4c4d8aa2accf2b2896c826150
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (ocefpaf) [inline recipes/geonum/meta.yaml]: Please use the sting from the example: https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L33

File: recipes/geonum/meta.yaml (hunk 1 above)
Comment (jgliss) [reply recipes/geonum/meta.yaml]: I fixed this in the latest commit.

File: recipes/geonum/meta.yaml (hunk 2 above)
Comment (jgliss) [reply recipes/geonum/meta.yaml]: Fixed!

**Comment by jgliss** [comment]
@conda-forge/staged-recipes I updated the requested changes and resubmitted the meta.yaml. All CI tests are passing. Could you please review once more? Thanks in advance!

**Pull request merged**